src/components/sections/Section/Section.tsx

```diff
@@ -0,0 +1,10 @@
+import './Section.scss'
```

Comment:
Hmm, could we change to the singular just to keep the pattern?
```suggestion
import './section.scss'
```